var/spack/repos/builtin/packages/py-ipython/package.py

```diff
@@ -39,12 +39,8 @@ class PyIpython(PythonPackage):
 
     depends_on('python@2.7:2.8,3.3:')
 
-    # These dependencies breaks concretization
-    # See https://github.com/spack/spack/issues/2793
-    # depends_on('py-backports-shutil-get-terminal-size', type=('build', 'run'), when="^python@:3.2")  # noqa
-    # depends_on('py-pathlib2', type=('build', 'run'), when="^python@:3.3")
-    depends_on('py-backports-shutil-get-terminal-size', type=('build', 'run'))
-    depends_on('py-pathlib2',                   type=('build', 'run'))
+    depends_on('py-backports-shutil-get-terminal-size', type=('build', 'run'), when="^python@:3.2")  # noqa
+    depends_on('py-pathlib2', type=('build', 'run'), when="^python@:3.3")
```

Comment:
> `when="^python@:3.2"`
> `when="^python@:3.3"`

if I remember well this only works when we append .99 as patch version, like you have for node.js. Or is there I understood wrong?

Reply:
I believe neurodamus needs mpi by default, unless we disable mpi in the whole hdf5 stack.

Reply:
> > `when="^python@:3.2"`
> > `when="^python@:3.3"`
> 
> if I remember well this only works when we append .99 as patch version, like you have for node.js. Or is there I understood wrong?

I don't think so..that just version specifier..it says `from_version:to_version`. People add `:3.1.99` instead of `:3.2` because in later case you `include` 3.2 as well.

Reply:
> I believe neurodamus needs mpi by default, unless we disable mpi in the whole hdf5 stack.

I am thinking now whether I should change default in py-h5py or not. Neurodamus doesn't use h5py and hence it should be fine.

Reply:
The python version currently in test uses it :/

Reply:
but do they need `py-h5py+mpi` ?